.circleci/config.yml

```diff
@@ -1,11 +1,11 @@
 version: 2.1
 orbs:
-  python: circleci/python@0.2.1
+  python: circleci/python@1.4.0
```

Comment:
I dont think the orb is actually used - could try removing it and seeing if the build still runs

Reply:
CI did not like that, apparently it is being used so I put it back.